static/src/js/components/Tree/TreeItem.js

```diff
@@ -132,7 +132,7 @@ export const TreeItem = ({
       className={treeItemClasses}
     >
       <div
-        className="tree-item__header"
+        className="tree-item__headervariable"
```

Comment:
nitpick kebab case for our scss classNames;
```suggestion
        className="tree-item__header-variable"
```